### DISCUSSION TITLE
Plugin options is not accessible using geth

### DISCUSSION THREAD

**Discussion by trinayanbhatt** [ConsenSys/quorum#1633] [Q&A]
I have installed the last version for geth 1.11.5 as well as some other previous versions but the support for 

`geth account **plugin** ` 

command is not available for standalone geth clients.

Although plugin method and commands for using quorum plugins is available in docker based geth client. 

Can someone please help in letting me know how can we enable **plugin** method or command for standalone **geth** clients?

**Comment by antonydenyer** [comment]
Quorum is a fork of geth. You need to raise this over at https://github.com/ethereum/go-ethereum/
